info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:31:13.067Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:31:22.542Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:32:01.588Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:32:10.025Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:32:19.315Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:49:31.932Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:49:36.182Z"}
error: 
    status - 500 
    message - Error: User already register with this email!
    at _callee2$ (E:\New folder\bookstore\src\services\/user.service.js:35:11)
    at tryCatch (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T04:50:34.723Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:52:38.200Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:52:47.938Z"}
error: 
      status - 8000
      message - cannot create a new database -- already using 100 databases of 100 
      url - /api/v1/users/registerUser 
      method - POST 
      IP - ::1
     {"timestamp":"2023-03-02T04:54:18.541Z"}
error: 
    status - 500 
    message - Error: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:400:5)
    at ServerResponse.setHeader (node:_http_outgoing:663:11)
    at ServerResponse.header (E:\New folder\bookstore\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (E:\New folder\bookstore\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (E:\New folder\bookstore\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (E:\New folder\bookstore\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (E:\New folder\bookstore\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (E:\New folder\bookstore\node_modules\express\lib\router\index.js:326:13)
    at E:\New folder\bookstore\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\New folder\bookstore\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (E:\New folder\bookstore\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (E:\New folder\bookstore\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:473:21) 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T04:54:18.622Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:57:12.766Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:57:21.732Z"}
error: 
      status - 8000
      message - cannot create a new database -- already using 100 databases of 100 
      url - /api/v1/users/registerUser 
      method - POST 
      IP - ::1
     {"timestamp":"2023-03-02T04:57:31.162Z"}
error: 
    status - 500 
    message - Error: Cannot set headers after they are sent to the client
    at new NodeError (node:internal/errors:400:5)
    at ServerResponse.setHeader (node:_http_outgoing:663:11)
    at ServerResponse.header (E:\New folder\bookstore\node_modules\express\lib\response.js:794:10)
    at ServerResponse.send (E:\New folder\bookstore\node_modules\express\lib\response.js:174:12)
    at ServerResponse.json (E:\New folder\bookstore\node_modules\express\lib\response.js:278:15)
    at appErrorHandler (E:\New folder\bookstore\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (E:\New folder\bookstore\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (E:\New folder\bookstore\node_modules\express\lib\router\index.js:326:13)
    at E:\New folder\bookstore\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (E:\New folder\bookstore\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (E:\New folder\bookstore\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (E:\New folder\bookstore\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:473:21) 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T04:57:31.229Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:58:48.539Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:58:52.669Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:59:13.961Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T04:59:37.963Z"}
info: Connected to the database. {"timestamp":"2023-03-02T04:59:42.102Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T05:00:08.993Z"}
info: Connected to the database. {"timestamp":"2023-03-02T05:00:18.841Z"}
error: 
    status - 500 
    message - Error: User already register with this email!
    at _callee2$ (E:\New folder\bookstore\src\services\/user.service.js:35:11)
    at tryCatch (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T05:00:51.979Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T05:03:10.344Z"}
info: Connected to the database. {"timestamp":"2023-03-02T05:03:15.580Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T08:15:31.348Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:351:10\n    at E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (E:\\New folder\\bookstore\\src\\config\\/database.js:12:20)\n    at tryCatch (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:125:22)\n    at Generator.next (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12","timestamp":"2023-03-02T08:16:04.177Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T08:22:03.346Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:351:10\n    at E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (E:\\New folder\\bookstore\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (E:\\New folder\\bookstore\\src\\config\\/database.js:12:20)\n    at tryCatch (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:44:17)\n    at Generator.<anonymous> (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:125:22)\n    at Generator.next (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:69:21)\n    at asyncGeneratorStep (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at E:\\New folder\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12","timestamp":"2023-03-02T08:22:33.311Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T08:24:12.569Z"}
info: Connected to the database. {"timestamp":"2023-03-02T08:24:12.623Z"}
error: 
    status - 500 
    message - TypeError: (0 , _user2.forgetemail) is not a function
    at _callee3$ (E:\New folder\bookstore\src\services\/user.service.js:51:40)
    at tryCatch (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (E:\New folder\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgetPassword 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T08:27:41.092Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T09:15:12.803Z"}
info: Connected to the database. {"timestamp":"2023-03-02T09:15:12.878Z"}
error: 
    status - 500 
    message - ValidationError: "email" must be a valid email 
    url - /api/v1/users/registerUser 
    method - POST 
    IP - ::1
   {"timestamp":"2023-03-02T09:16:32.568Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T09:21:50.753Z"}
info: Connected to the database. {"timestamp":"2023-03-02T09:21:50.813Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T09:24:19.032Z"}
info: Connected to the database. {"timestamp":"2023-03-02T09:24:19.091Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T09:46:26.481Z"}
info: Connected to the database. {"timestamp":"2023-03-02T09:46:26.520Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T09:47:02.426Z"}
info: Connected to the database. {"timestamp":"2023-03-02T09:47:02.466Z"}
error: uncaughtException: noteRoute is not defined
ReferenceError: noteRoute is not defined
    at routes (E:\New folder\bookstore\src\routes\/index.js:15:28)
    at Object.<anonymous> (E:\New folder\bookstore\src\/index.js:32:38)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._compile (E:\New folder\bookstore\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Object.newLoader [as .js] (E:\New folder\bookstore\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Function.Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.runMain (node:internal/modules/run_main:82:12)
    at Object.<anonymous> (E:\New folder\bookstore\node_modules\@babel\node\src\_babel-node.ts:223:12)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Function.Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)
    at node:internal/main/run_main_module:23:47 {"date":"Thu Mar 02 2023 15:20:37 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":672433.656},"process":{"argv":["node","E:\\New folder\\bookstore\\src\\index.js"],"cwd":"E:\\New folder\\bookstore","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18377486,"external":21712151,"heapTotal":124661760,"heapUsed":90572448,"rss":168189952},"pid":9292,"uid":null,"version":"v19.3.0"},"stack":"ReferenceError: noteRoute is not defined\n    at routes (E:\\New folder\\bookstore\\src\\routes\\/index.js:15:28)\n    at Object.<anonymous> (E:\\New folder\\bookstore\\src\\/index.js:32:38)\n    at Module._compile (node:internal/modules/cjs/loader:1218:14)\n    at Module._compile (E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)\n    at Object.newLoader [as .js] (E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1081:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:922:12)\n    at Function.runMain (node:internal/modules/run_main:82:12)\n    at Object.<anonymous> (E:\\New folder\\bookstore\\node_modules\\@babel\\node\\src\\_babel-node.ts:223:12)\n    at Module._compile (node:internal/modules/cjs/loader:1218:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1272:10)\n    at Module.load (node:internal/modules/cjs/loader:1081:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:922:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)\n    at node:internal/main/run_main_module:23:47","timestamp":"2023-03-02T09:50:37.633Z","trace":[{"column":28,"file":"E:\\New folder\\bookstore\\src\\routes\\/index.js","function":"routes","line":15,"method":null,"native":false},{"column":38,"file":"E:\\New folder\\bookstore\\src\\/index.js","function":null,"line":32,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1218,"method":"_compile","native":false},{"column":24,"file":"E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1272,"method":".js","native":false},{"column":7,"file":"E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1081,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":922,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.runMain","line":82,"method":"runMain","native":false},{"column":12,"file":"E:\\New folder\\bookstore\\node_modules\\@babel\\node\\src\\_babel-node.ts","function":null,"line":223,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1218,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1272,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1081,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":922,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":82,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":23,"method":null,"native":false}]}
error: uncaughtException: bookstoreRoute is not defined
ReferenceError: bookstoreRoute is not defined
    at routes (E:\New folder\bookstore\src\routes\/index.js:15:28)
    at Object.<anonymous> (E:\New folder\bookstore\src\/index.js:32:38)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._compile (E:\New folder\bookstore\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Object.newLoader [as .js] (E:\New folder\bookstore\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Function.Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.runMain (node:internal/modules/run_main:82:12)
    at Object.<anonymous> (E:\New folder\bookstore\node_modules\@babel\node\src\_babel-node.ts:223:12)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Function.Module._load (node:internal/modules/cjs/loader:922:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)
    at node:internal/main/run_main_module:23:47 {"date":"Thu Mar 02 2023 15:35:18 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":673314.875},"process":{"argv":["node","E:\\New folder\\bookstore\\src\\index.js"],"cwd":"E:\\New folder\\bookstore","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19340541,"external":21694970,"heapTotal":123613184,"heapUsed":100249016,"rss":168148992},"pid":7152,"uid":null,"version":"v19.3.0"},"stack":"ReferenceError: bookstoreRoute is not defined\n    at routes (E:\\New folder\\bookstore\\src\\routes\\/index.js:15:28)\n    at Object.<anonymous> (E:\\New folder\\bookstore\\src\\/index.js:32:38)\n    at Module._compile (node:internal/modules/cjs/loader:1218:14)\n    at Module._compile (E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)\n    at Object.newLoader [as .js] (E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:1081:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:922:12)\n    at Function.runMain (node:internal/modules/run_main:82:12)\n    at Object.<anonymous> (E:\\New folder\\bookstore\\node_modules\\@babel\\node\\src\\_babel-node.ts:223:12)\n    at Module._compile (node:internal/modules/cjs/loader:1218:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1272:10)\n    at Module.load (node:internal/modules/cjs/loader:1081:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:922:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:82:12)\n    at node:internal/main/run_main_module:23:47","timestamp":"2023-03-02T10:05:18.847Z","trace":[{"column":28,"file":"E:\\New folder\\bookstore\\src\\routes\\/index.js","function":"routes","line":15,"method":null,"native":false},{"column":38,"file":"E:\\New folder\\bookstore\\src\\/index.js","function":null,"line":32,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1218,"method":"_compile","native":false},{"column":24,"file":"E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1272,"method":".js","native":false},{"column":7,"file":"E:\\New folder\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1081,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":922,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.runMain","line":82,"method":"runMain","native":false},{"column":12,"file":"E:\\New folder\\bookstore\\node_modules\\@babel\\node\\src\\_babel-node.ts","function":null,"line":223,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1218,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1272,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":1081,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":922,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":82,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":23,"method":null,"native":false}]}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T10:09:42.329Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T10:09:47.943Z"}
info: Connected to the database. {"timestamp":"2023-03-02T10:09:47.978Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T10:11:01.685Z"}
info: Connected to the database. {"timestamp":"2023-03-02T10:11:01.719Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T10:12:27.151Z"}
info: Connected to the database. {"timestamp":"2023-03-02T10:12:27.190Z"}
info: Server started at http://localhost:5455/api/v1/ {"timestamp":"2023-03-02T10:22:55.487Z"}
info: Connected to the database. {"timestamp":"2023-03-02T10:22:55.534Z"}
